include/vkgcDefs.h

```diff
@@ -605,7 +605,8 @@ struct ShaderModuleUsage {
   bool enableRayQuery;     ///< Whether the "RayQueryKHR" capability is used
   bool rayQueryLibrary;    ///< Whether the shaderModule is rayQueryLibrary
   bool isInternalRtShader; ///< Whether the shaderModule is a GPURT internal shader (e.g. BVH build)
-  bool hasTraceRay;        ///< Whether the shaderModule has OpTraceRayKHR;
+  bool hasTraceRay;        ///< Whether the shaderModule has OpTraceRayKHR
+  bool hasExecuteCallable; ///< Whether the shaderModule has OpExecuteCallableKHR
```

Comment:
whereis this bool used in the llpcCompiling if  shaderModuleUsage hasExecuteCallable?

Reply:
It isn't, at least not yet. There is a check in SPIRVReader that forces indirect compile mode when OpExecuteCallable is used. The idea was that this will allow us to move this check outside and eventually run SPIRVReader in helper threads.